src/cli/Create.cpp

```diff
@@ -69,6 +78,23 @@ int Create::execute(const QStringList& arguments)
         return EXIT_FAILURE;
     }
 
+    // Validate the decryption time before asking for a password.
```

Comment:
@droidmonkey @phoerious I'm not sure about this one. Should we make sure the CLI consumes the password even if an error is raised, in case the password is piped into the process?

Reply:
I agree with validating all parameters before asking for sensitive information like the password.